inscriptions/ISic003469.xml

```diff
@@ -187,87 +192,23 @@
-                    <expan>
-                        <abbr>M</abbr>
-                        <ex>arci</ex>
-                    </expan>
-                    Lívi
-                    <expan>
-                        <abbr>M</abbr>
-                        <ex>arci</ex>
-                    </expan>
-                    <expan>
-                        <abbr>f</abbr>
-                        <ex>ili</ex>
-                    </expan>
-                    <expan>
-                        <abbr>Qui</abbr>
-                        <ex>rina</ex>
-                    </expan>
-                    Optati flaminis G<supplied reason="lost">a</supplied>u<unclear>l</unclear>(itanorum)<lb n="7"/>
-                    Iuliae Augusti
-                    <del rend="erasure">
-                        <expan>
-                            <abbr>Imp</abbr>
-                            <ex>eratoris</ex>
-                        </expan>
-                        <expan>
-                            <abbr>perpet</abbr>
-                            <ex>ui</ex>
-                        </expan>
-                    </del>
-                    cum
-                    <unclear>V</unclear>
-                    <lb n="8"/>líberis
-                    <expan>
-                        <abbr>s</abbr>
-                        <ex>ua</ex>
-                    </expan>
-                    <expan>
-                        <abbr>p</abbr>
-                        <ex>ecunia</ex>
-                    </expan>
-                    consacravit.
+                    <lb n="1"/>Cereri <g ref="#interpunct">·</g> <name>Iuliae</name> <g ref="#interpunct">·</g> <name>Augustae</name>
+                    <lb n="2"/>Divi <g ref="#interpunct">·</g> Augusti <g ref="#interpunct">·</g> Matri
+                    <lb n="3"/><expan><abbr>Ti</abbr><ex>beri</ex></expan> <g ref="#interpunct">·</g> Caesaris <g ref="#interpunct">·</g> Augusti
+                    <lb n="4"/><name>Lutatia</name> <g ref="#interpunct">·</g> <name><expan><abbr>C</abbr><ex>ai</ex></expan></name> <g ref="#interpunct">·</g> <expan><abbr>f</abbr><ex>ilia</ex></expan> <g ref="#interpunct">·</g> sacerdos <g ref="#interpunct">·</g> Augustae
+                    <lb n="5"/><add place="overstrike"><expan><abbr>Imp</abbr><ex>eratoris</ex></expan> <expan><abbr>perpet</abbr><ex>ui</ex></expan></add> uxor
+                    <lb n="6"/><name><expan><abbr>M</abbr><ex>arci</ex></expan> <g ref="#interpunct">·</g> Livi <g ref="#interpunct">·</g></name> <name><expan><abbr>M</abbr><ex>arci</ex></expan></name> <g ref="#interpunct">·</g> <expan><abbr>f</abbr><ex>ili</ex></expan> <g ref="#interpunct">·</g> <expan><abbr>Qui</abbr><ex>rina</ex></expan> <g ref="#interpunct">·</g> <name>Optati</name> <g ref="#interpunct">·</g> flaminis <g ref="#interpunct">·</g> <expan><ex>Gaul</ex><abbr>itanorum</abbr></expan> <g ref="#interpunct">·</g> <!--for some reason I can't seem to add a supplied reason="lost" for the 'a' within Gaul in the expansion-->
```

Comment:
put the name tag around each individual name element only (and not the interpuncts).

Reply:
Your problem with G[a]ul(itanorum) at the end is due to the fact that you've inverted the `<ex>` and `<abbr>` tags, and the former can't take supplied, because it's already an editorial expansion.